lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -210,17 +211,26 @@ def test_no_ccx_created(self):
     def test_create_ccx(self):
         """
         Create CCX. Follow redirect to coach dashboard, confirm we see
-        the coach dashboard for the new CCX.
+        the coach dashboard for the new CCX. Also, confirm that the coach
+        has staff access to it.
         """
+
         self.make_coach()
         url = reverse(
             'create_ccx',
             kwargs={'course_id': self.course.id.to_deprecated_string()})
+
         response = self.client.post(url, {'name': 'New CCX'})
         self.assertEqual(response.status_code, 302)
         url = response.get('location')  # pylint: disable=no-member
         response = self.client.get(url)
         self.assertEqual(response.status_code, 200)
+        pattern = "http://[^\/]+/courses/([^\/]+)/ccx_coach"
+        results = re.match(pattern, url)
```

Comment:
Ouch. I try to avoid regexes unless absolutely necessary - why do you need it here? I'm shocked you can't get the course id from the url in some other way